googlemock/cmake/Config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
+check_required_components("@PROJECT_NAME@")
```

Comment:
For cmake I mainly use Notepad++ and sometimes Visual Studio. I will add a newline.